packages/liferay-npm-scripts/src/utils/spawnSync.js

```diff
@@ -17,8 +17,9 @@ function getDescription(command, args) {
  * @param {string} command Path to bin file
  * @param {Array=} args=[] List of string arguments
  * @param {Object=} options={} Options to pass to spawn.sync
+ * @param {boolean} silent=false Should swallow non-error exit codes.
  */
-module.exports = function(command, args = [], options = {}) {
+module.exports = function(command, args = [], options = {}, silent = false) {
```

Comment:
If you want to add another option, it should go in the `options` object rather than being tagged on the end. You can "pluck" it out from the others with destructuring:

```javascript
const {silent, ...spawnOptions} = options;
```

Having said that, I think what we really want here is still to exit with a non-zero status (because that's the expected behavior in any of these tools) but we want the output to be less noisy, which we can achieve by printing the exception message only rather than the entire stack trace, which is the ugly bit.